specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -0,0 +1,304 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "AttestationClient",
+        "description": "Describes the interface for the per-tenant enclave service.",
+        "version": "2018-09-01-preview"
+        
+    },   
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+    "paths": {
+      "/operations/update": {
+            "post": {
+                "tags": [
+                         "attestation",
+                         "policy"
+                        ],
+                "summary": "Accepts a new policy document and returns a JWT which expresses  used in preparation to set attestation policy.",
+                "operationId": "PrepareToSetPolicy",
+                "consumes": [ "text/plain" ],
+                "produces": [ "text/plain", "application/json" ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AuthorizationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/TeeKind"
+                    },
+                    {
+                        "name": "PolicyJws",
+                        "in": "body",
+                        "description": "JSON Web Signature (See RFC7515) expressing the new policy",
+                        "required": true,
+                        "x-ms-client-flatten": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                  "200": {
+                    "description": "Success - Returns a JWT signed by the metadata signing key that contains the hash of the supplied policy to be set.",
+                    "schema": {
+                      "$ref": "#/definitions/AttestationPolicy"
+                    }
+                  },
+                    "400": {
```

Comment:
normally not define error code in swagger. and pls define "default" response

Reply:
default response has been added.